spec/models/book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe 'Book' do
+  subject(:unLibro) { Book.new }
+
+  context 'With an empty book' do
+    it '#attr_getters' do
+      %w[genre author image title publisher year].each do |un_atr|
+        expect(unLibro.attributes.keys).to include(un_atr)
```

Comment:
hmmm I'm not sure what are you trying to do here 🤔 I would add [shoulda_matchers](https://github.com/thoughtbot/shoulda-matchers) to validate the model's validations.

If you have any doubt about how to write a test, remember to check our [style guide](https://github.com/Wolox/tech-guides/blob/master/ruby-on-rails/docs/standards/rspec-style-guide.md)!